### PR TITLE
Doxygen for getgbe.f getgbeh.f getgbem.f getgbemh.f getgbemn.f (Part of #77)

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
Doxygen work for the following:

getgbe.f
getgbeh.f
getgbem.f
getgbemh.f
getgbemn.f

Part of #77

### CODE REVIEW

File: src/getgbeh.f
Comment (edwardhartnett) [inline src/getgbeh.f]: Put a space between the C> and the start of text. So:

`C> Read a grib index file (or optionally the grib file itself) to get
`

**Review by edwardhartnett — CHANGES_REQUESTED** [review]

**Review by edwardhartnett — APPROVED** [review]

**Review by edwardhartnett — APPROVED** [review]

**Comment by BrianCurtis-NOAA** [comment]
@Hang-Lei-NOAA waiting on your approval before I merge.

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Pull request merged**
